.github/workflows/readme.yml

```diff
@@ -0,0 +1,33 @@
+name: Translate README
+
+on:
+  push:
+    branches:
+      - main
+      - master
+      - auto-translation
```

Comment:
Hi，I want to confirm whether the automatic translation is triggered every time the main branch changes, or only when the readme file changes？